src/main/java/duke/models/assignedtasks/AssignedTaskManager.java

```diff
@@ -29,6 +31,7 @@ public AssignedTaskManager(ArrayList<AssignedTask> assignedTasks) {
                 assignedTask.setUuid(maxId);
```

Comment:
There are some problem in AssignedTaskManager’s Data structure. When you are trying to load a arraylist of AssignedTask into the AssignedTaskManager, you set the current maxId(unique Is of the task) as the last element in the Multimaps’s unique Id+1 , however this multimap’s key is not this unique Id but it’s patient Id. Therefore the last element’s unique id is not the actual maxId.

Reply:
@qjie7 Nice catch. It makes sense. 
@WEIFENG-NUSCEG During loading/saving, the records are listed in according to the patientID. Thus your maxID may not be your maxUuid.

Reply:
Hi thanks for the catch, the problem has been solved in the new commit.